CMakeLists.txt

```diff
@@ -321,8 +323,18 @@ if(${HAVE_BABELTRACE})
   set(WITH_BABELTRACE ${BABELTRACE_FOUND})
   set(HAVE_BABELTRACE_BABELTRACE_H ${BABELTRACE_FOUND})
   set(HAVE_BABELTRACE_CTF_EVENTS_H ${BABELTRACE_FOUND})
+  set(HAVE_BABELTRACE_CTF_ITERATOR_H ${BABELTRACE_FOUND})
 endif(${HAVE_BABELTRACE})
 
+option(DEBUG_GATHER "C_Gather debugging is enabled" OFF)
+option(HAVE_KINETIC "Kinetic Client is enables" OFF)
+option(HAVE_LIBZFS "LibZFS is enabled" OFF)
```

Comment:
@liewegas for now I just set this option to OFF to let the user set the variable at the command line. Should I create a Findzfs.cmake for this?